### ISSUE TITLE
download link for influxdb-enterprise web console results in 403

### ISSUE THREAD

**Issue by paulyea** [influxdata/docs.influxdata.com#877]
Following the direction for installing the web console for influx enterprise 1.1 results in a 403:
```
wget https://s3.amazonaws.com/influx-enterprise/releases/influx-enterprise_1.1.0_amd64.deb
--2016-11-14 15:25:12--  https://s3.amazonaws.com/influx-enterprise/releases/influx-enterprise_1.1.0_amd64.deb
Resolving s3.amazonaws.com... 52.216.64.19
Connecting to s3.amazonaws.com|52.216.64.19|:443... connected.
HTTP request sent, awaiting response... 403 Forbidden
2016-11-14 15:25:13 ERROR 403: Forbidden.
```
###### URL for relevant page?
http://docs.influxdata.com/enterprise/v1.1/quickstart_installation/web_console_installation/
https://github.com/influxdata/docs.influxdata.com/blob/master/content/enterprise/v1.1/production_installation/web_console_installation.md
###### What products and version are you using?
influx enterprise 1.1.0
###### Where did you look before opening the issue?
current open issues: https://github.com/influxdata/docs.influxdata.com/issues
as well as https://github.com/influxdata/docs.influxdata.com/blob/master/CONTRIBUTING.md

**Comment by rkuchan** [comment]
@paulyea, the docs seem to be ahead of the actual build. I've alerted the team - while we get this fixed you can download and install version 1.0.3 of the web console:

```
wget https://s3.amazonaws.com/influx-enterprise/releases/influx-enterprise_1.0.3_amd64.deb
sudo dpkg -i influx-enterprise_1.0.3_amd64.deb
```

Apologies for the confusion. I'll keep this open until it gets resolved by the team.

**Comment by rkuchan** [comment]
@paulyea, the 1.1 build for the web console is available. Apologies for the delay.
